src/client/pfs/pfs.proto

```diff
@@ -307,16 +307,15 @@ enum Delimiter {
   CSV = 4;
 }
 
-message AppendFileRequest {
+message AppendFile {
   bool overwrite = 1;
   string tag = 2;
```

Comment:
In the storage layer I think you can do a tag per file. Should this be in the sources?

Reply:
An append file modification will correspond to one file, a tar stream, or a url. If you want to add different tags for a file (or set of files), then those should be separate append file modifications.